src/ngx_http_lua_pipe.c

```diff
@@ -572,6 +572,15 @@ ngx_http_lua_ffi_pipe_spawn(ngx_http_lua_ffi_pipe_proc_t *proc,
     ngx_http_lua_pipe_signal_t     *sig;
     sigset_t                        set;
 
+#if !(NGX_HTTP_LUA_HAVE_EXECVPE)
+    if (environ != NULL) {
+        *errbuf_size = ngx_snprintf(errbuf, *errbuf_size,
+                                    "environ option unsupported")
```

Comment:
The following is more explicit and aligned with existing similar error messages: 
```
environ option not supported
```